Assets/VRTK/SDK/HyperealVR/SDK_HyperealVRController.cs

```diff
@@ -0,0 +1,553 @@
+﻿// HyperealVR Controller|SDK_HyperealVR|004
+namespace VRTK
+{
+#if VRTK_DEFINE_SDK_HYPEREALVR
+    using UnityEngine;
+    using System.Collections.Generic;
+    using Hypereal;
+    using System;
+#endif
+
+    /// <summary>
+    /// The HyperealVR Controller SDK script provides a bridge to SDK methods that deal with the input devices.
+    /// </summary>
+    [SDK_Description(typeof(SDK_HyperealVRSystem))]
+    public class SDK_HyperealVRController
+#if VRTK_DEFINE_SDK_HYPEREALVR
+        : SDK_BaseController
+#else
+        : SDK_FallbackController
+#endif
+    {
+#if VRTK_DEFINE_SDK_HYPEREALVR
+        private VRTK_TrackedController cachedLeftController;
+        private VRTK_TrackedController cachedRightController;
+        private VRTK_TrackedController cachedLeftTrackedObject;
+        private VRTK_TrackedController cachedRightTrackedObject;
+
+        /// <summary>
+        /// The ProcessUpdate method enables an SDK to run logic for every Unity Update
+        /// </summary>
+        /// <param name="controllerReference">The reference for the controller.</param>
+        /// <param name="options">A dictionary of generic options that can be used to within the update.</param>
+        public override void ProcessUpdate(VRTK_ControllerReference controllerReference, Dictionary<string, object> options)
+        {
+        }
+
+        /// <summary>
+        /// The ProcessFixedUpdate method enables an SDK to run logic for every Unity FixedUpdate
+        /// </summary>
+        /// <param name="controllerReference">The reference for the controller.</param>
+        /// <param name="options">A dictionary of generic options that can be used to within the fixed update.</param>
+        public override void ProcessFixedUpdate(VRTK_ControllerReference controllerReference, Dictionary<string, object> options)
+        {
+        }
+
+        /// <summary>
+        /// The GetCurrentControllerType method returns the current used ControllerType based on the SDK and headset being used.
+        /// </summary>
+        /// <returns>The ControllerType based on the SDK and headset being used.</returns>
+        public override ControllerType GetCurrentControllerType()
+        {
+            return ControllerType.Custom;
+        }
+
+        /// <summary>
+        /// The GetControllerDefaultColliderPath returns the path to the prefab that contains the collider objects for the default controller of this SDK.
+        /// </summary>
+        /// <param name="hand">The controller hand to check for</param>
+        /// <returns>A path to the resource that contains the collider GameObject.</returns>
+        public override string GetControllerDefaultColliderPath(ControllerHand hand)
+        {
+            return "ControllerColliders/Fallback";
```

Comment:
Isn't there a Hyperreal controller? the fallback collider is just a sphere, is that good enough for the Hyperreal experience?